### ISSUE TITLE
invalid querystring version with 5.0.7 under package.json

### ISSUE THREAD

**Issue by Azerothian** [SocketCluster/socketcluster-client#61]
https://www.npmjs.com/package/querystring

```
npm ERR! Darwin 15.5.0                                                                                                                              
npm ERR! argv "/usr/local/Cellar/node/6.5.0/bin/node" "/usr/local/bin/npm" "install"                                                                
npm ERR! node v6.5.0                                                                                                                                
npm ERR! npm  v3.10.6                                                                                                                               
                                                                                                                                                    
npm ERR! No compatible version found: querystring@1.2.0                                                                                             
npm ERR! Valid install targets:                                                                                                                     
npm ERR! 0.2.0, 0.1.0, 0.0.4, 0.0.1                                                                                                                 
npm ERR!                                                                                                                                            
npm ERR!                                                                                                                                            
npm ERR! If you need help, you may report this error at:                                                                                            
npm ERR!     <https://github.com/npm/npm/issues>                                                                                                    
                                                                                                                                                    
npm ERR! Please include the following file with any support request:                                                                                
npm ERR!     /Users/user/projects/scchat/npm-debug.log
```

**Comment by hustcer** [comment]
+1, same here

**Comment by zalmoxisus** [comment]
Fixed in #60.

**Comment by jondubois** [comment]
My bad - I messed up the version number. Fix has been published to npm.
